cloudos_cli/clos.py

```diff
@@ -607,10 +639,7 @@ def process_project_list(r, all_fields=False):
                    'jobCount',
                    'notebookSessionCount'
                    ]
-        my_projects = json.loads(r.content)
-        if 'projects' in my_projects:
-            my_projects = my_projects['projects']
-        df_full = pd.json_normalize(my_projects)
+        df_full = pd.json_normalize(r)
```

Comment:
@danielboloc There is an issue here, that is also present in the `process_job_list` method. When the user selects a page that does not exist, like `--page 10000` , if they want the CSV output, so if this function runs, it fails. The problem is that the API is returning an empty response in such a case. Could you please add the following here (and in `process_job_list` to prevent the error?
```suggestion
        df_full = pd.json_normalize(r)
        if df_full.empty:
            return df_full
```
```